scripts/cipublish

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GT_CHATTA_DEBUG}" ]]; then
+    set -x
+fi
+
+if [[ -n "${TRAVIS_COMMIT}" ]]; then
+    VERSION="${TRAVIS_COMMIT:0:7}"
+else
+    VERSION="$(git rev-parse --short HEAD)"
+fi
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+Publish images to Amazon ECR
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+        if [[ -n "${AWS_ECR_ENDPOINT}" ]]; then
+
+            echo "Building geotrellis-service JAR..."
+            docker-compose run --rm gt-chatta assembly
+
+            VERSION="${VERSION}" \
+            docker-compose -f docker-compose.yml \
+                -f docker-compose.test.yml build \
+                gt-chatta
+
+            # aws ecr get-login includes a deprecated flag
+            eval $(aws ecr get-login | sed "s/-e none//")
```

Comment:
With newer versions of the AWS CLI, we should be able to get around this with something like https://github.com/azavea/raster-foundry/commit/c1e568614097cc2f74deb1b806627bc6b26295d3.